apps/ui/src/views/Proposal.vue

```diff
@@ -191,6 +200,44 @@ watchEffect(() => {
                 </UiButton>
               </UiTooltip>
             </div>
+            <div v-else-if="proposal.type === 'single-choice'" class="flex flex-col space-y-2 py-2">
+              <UiButton
+                v-for="(choice, index) in proposal.choices"
+                :key="index"
+                class="!h-[48px] text-left w-full truncate"
+                :loading="sendingType === index + 1"
+                @click="handleVoteClick(index + 1)"
+              >
+                {{ choice }}
+              </UiButton>
+            </div>
+            <div v-else-if="proposal.type === 'approval'" class="flex flex-col space-y-3 py-2">
+              <div class="flex flex-col space-y-2">
+                <UiButton
```

Comment:
Keyboard focus ring looks very weird on the choice buttons, probably cause by using `!h-[48px]`

Reply:
<img width="314" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/51686767/34c09c54-d6e2-436c-8b7a-045ed6a0a8b4">